packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -188,7 +189,8 @@ TaskInfos.propTypes = {
 
 const VmTask = ({ children, className, restartVmJob, task }) => (
   <li className={className}>
-    <Vm id={task.data.id} name={task.data.name_label} link newTab /> <TaskStateInfos status={task.status} />{' '}
+    <Vm id={task.data.id} name={task.data.name_label ?? VM_NOT_PRESENT} link newTab />{' '}
```

Comment:
I'm not a fan of this change:

- `task.data.name_label` should provide the name of the VM, maybe there is an issue in the backup code
- `VM_NOT_PRESENT` is not a VM name, semantically it makes no sens to pass this the `name` property of the `Vm` component
- this message could be displayed for multiple reasons and I'm not sure it is always appropriate, examples:
  -  VM no longer exists
  - objets are not loaded yet in xo-web
  - the pool is no longer connected in xo-server

For now I think it would be best to remove this change and merge only the other fix.